scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Scheduler.java

```diff
@@ -1737,4 +1737,26 @@ TaskStatesPage getTaskPaginated(String jobId, String statusFilter, int offset, i
      */
     Map<String, Map<String, Boolean>> checkJobsPermissionMethods(List<String> jobIds, List<String> methods)
             throws NotConnectedException, UnknownJobException;
+
+    /**
+     * Add an external endpoint url to the endpoint list of the considered job
```

Comment:
Attach an external endpoint url to a job. The job must not be terminated.

Reply:
Ok